app/controllers/arts_controller.rb

```diff
@@ -2,8 +2,12 @@ class ArtsController < ApplicationController
   skip_before_action :authenticate_user!, only: %i[index show]
 
   def index
-    @arts = Art.all
     @categories = Category.all
+    if params[:query].present?
+      @arts = Art.global_search(params[:query])
+    else
+      @arts = Art.all
+    end
```

Comment:
Can you rename "query" as "search" ? I feel like that would be more understandable

Reply:
What do you mean?

Reply:
The params :query, can you rename it as :search ?

Reply:
doesn't work if I do

Reply:
Also I think it's standard to call it query

Reply:
You need to also call it :search in the html
```
<%= text_field_tag :search,
      params[:search],
      placeholder: "Find an art piece"
    %>
```

Is it standard ? I've always noticed the param being called some form of "search" when I'm on website and from my quick research there doesn't really seem to be any standard really 🤔 
![image](https://user-images.githubusercontent.com/75388869/221529100-aa139030-09b5-4969-948f-f1af15bd5bfa.png)
![image](https://user-images.githubusercontent.com/75388869/221529129-c1c4b6d4-9774-4cfa-aea9-f846136770c8.png)
![image](https://user-images.githubusercontent.com/75388869/221529177-57df71e5-7571-4165-ac3c-2164186d70b7.png)

Reply:
ah you are right, I checked google but didn't see the search word but it's there.

changed it now.